WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
Cool, glad we're on the same page, that was surprisingly difficult to put into words! Remaining options:

- Stick with `centiseconds`
- Change it to `attempt value`, which is already defined elsewhere in the WCIF
- Use a string, so `"1.23"` instead of `1.23` (I'm not willing to do this, just including it for completeness)